install/install.sh

```diff
@@ -12,6 +12,7 @@ set -e
 SUPPORTED_ARCHITECTURES=(
   "armhf" # Pi, Pi2, Pi3, Pi4
   "armv7" # Pi2, Pi3, Pi4
+  "armv7l" # Pi4 (Raspberry Pi OS Bullseye)
```

Comment:
Pi2 and Pi3 are both also ARMv8, so `armv7l` comment should also reflect it, or add it on the side of the `armv7` line.

Reply:
done